src/frontend/src/pages/WorkPackageDetailPage/StageGateWorkPackageModalContainer/StageGateWorkPackageModal.tsx

```diff
@@ -124,5 +97,4 @@ const StageGateWorkPackageModal: React.FC<StageGateWorkPackageModalProps> = ({ w
     </Dialog>
   );
 };
-
```

Comment:
For code formatting, make sure to have line space, including the above lines (lines 5, 25, 42) where you deleted. In other words, restore the deleted lines.

Reply:
On it!